content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -121,54 +131,60 @@ For example, if there are 1,253 pods on the cluster and the client wants to rece
 
 1. List all of the pods on a cluster, retrieving up to 500 pods each time.
 
-        GET /api/v1/pods?limit=500
-        ---
-        200 OK
-        Content-Type: application/json
-        {
-          "kind": "PodList",
-          "apiVersion": "v1",
-          "metadata": {
-            "resourceVersion":"10245",
-            "continue": "ENCODED_CONTINUE_TOKEN",
-            ...
-          },
-          "items": [...] // returns pods 1-500
-        }
+   ```console
+   GET /api/v1/pods?limit=500
+   ---
+   200 OK
+   Content-Type: application/json
+   {
+     "kind": "PodList",
+     "apiVersion": "v1",
+     "metadata": {
+       "resourceVersion":"10245",
+       "continue": "ENCODED_CONTINUE_TOKEN",
+       ...
+     },
+     "items": [...] // returns pods 1-500
+   }
+   ```
 
 2. Continue the previous call, retrieving the next set of 500 pods.
 
-        GET /api/v1/pods?limit=500&continue=ENCODED_CONTINUE_TOKEN
-        ---
-        200 OK
-        Content-Type: application/json
-        {
-          "kind": "PodList",
-          "apiVersion": "v1",
-          "metadata": {
-            "resourceVersion":"10245",
-            "continue": "ENCODED_CONTINUE_TOKEN_2",
-            ...
-          },
-          "items": [...] // returns pods 501-1000
-        }
+   ```console
+   GET /api/v1/pods?limit=500&continue=ENCODED_CONTINUE_TOKEN
+   ---
+   200 OK
+   Content-Type: application/json
```

Comment:
mini nit: HTTP/1.1 usually has a blank line between headers and body; I'd include that here (it looks odd to me without it)